src/amoc.app.src

```diff
@@ -13,7 +13,8 @@
                   ssl,
                   trails,
                   cowboy,
-                  cowboy_swagger
+                  cowboy_swagger,
+                  fusco
```

Comment:
If fusco is only used in the tests it doesn't have to be included in the app.src file.